modules/virt-general-availability.adoc

```diff
@@ -0,0 +1,8 @@
+// Module included in the following assemblies:
+//
+// * virt/virt-2-4-release-notes.adoc
+
+[id="virt-general-availability_{context}"]
+= General availability
+
+{VirtProductName} is now a fully supported and generally available feature of the OpenShift Container Platform 4.5. Previously known as container-native virtualization (CNV), OpenShift virtualization enables you to bring traditional virtual machines (VMs) into OpenShift where they run with containers, and are managed as native Kubernetes objects. 
```

Comment:
If `{VirtProductName} ` will include "Red Hat", then the use of `(CNV), OpenShift virtualization` is fine. If not, then `(CNV), OpenShift virtualization` needs to be chanegd to `(CNV), Red Hat OpenShift virtualization`

Reply:
* Actually "Red Hat" should precede the first usage of {VirtProductName}, which will build as 'OpenShift Virtualization' (so that it build as "Red Hat OpenShift Virtualization is now a fully..."). Our understanding is that the context of it being Red Hat OpenShift Container Platform is thought to be already established in the docs, however I agree with Fabian that this is a good place to reiterate it. The best place, perhaps :)
* Remove "(CNV)" as we do not refer to the product in the docs as CNV.
* 'OpenShift virtualization' should be changed to {VirtProductName} and "OpenShift Container Platform" should be {product-title}.
* "...into Openshift where they.." <-- OpenShift should be {product-title}. We don't call it OpenShift.

Reply:
Adding @sbream to this PR.